examples/vault-s3-backend/variables.tf

```diff
@@ -52,7 +52,7 @@ variable "vault_instance_type" {
 
 variable "consul_instance_type" {
   description = "The type of EC2 Instance to run in the Consul ASG"
-  default     = "t2.nano"
+  default     = "t2.micro"
```

Comment:
I had switched them to nano on previous PR, decided to switch them back to micro because errors caused by consul not being available started popping up, I didnt get to investigate origin in details because this improved it